includes/compatibility/class-cpt-compat-mode.php

```diff
@@ -139,6 +139,7 @@ public function get_title( $title, $id ) {
 		} elseif ( function_exists( 'avia_title' ) ) {
 			// Change the main title in Enfold theme.
 			add_filter( 'avf_title_args', array( &$this, 'enfold_title' ), 10, 2 );
+			$this->enfold_maybe_add_single_title();
```

Comment:
Is there a way we can handle this in the WPBDP__Themes_Compat class? Maybe we could use the avf_title_args filter in the line above instead of adding an extra function here?

Reply:
No, there isn't. because avf_title_args executes after the listing-title.tpl.php file.

Reply:
How about we add a new hook here instead then and move all of this theme code to the other class? Like this:

```
if ( is_main_query() && ( in_the_loop() || $is_block ) && $id === (int) wpbdp_get_page_id( 'main' ) ) {
			$title = $this->page_title;
		}
$title = apply_filters( 'wpbdp_get_title_compat', $title );
```

Then, inside `theme_enfold_before_dispatch()` we can add:
```
// Change the main title in Enfold theme.
add_filter( 'avf_title_args', array( &$this, 'enfold_title' ), 10, 2 );
add_filter( 'wpbdp_get_title_compat', 'new_function_here_that_returns_title...' );
```

Reply:
Please see the codes below!

![image](https://user-images.githubusercontent.com/69119241/211582107-cb124ed0-cc72-45bf-a73e-449878131d54.png)
![image](https://user-images.githubusercontent.com/69119241/211582206-2454bcd4-73d8-4fe3-abb3-79936cbb718f.png)


One of the files `class-themes-compat.php` or `class-themes-compat.php` is executed and we cannot connect the codes of these 2 classes together.